bin/install_ATArchiver.sh

```diff
@@ -0,0 +1,101 @@
+DOWNLOAD_DIR=~/downloads
+CTRL_IIP_PREFIX=/home/ATS/src/git 
+
+yum update -y && yum install -y cmake make gcc-c++ openssl openssl-devel
+useradd ATS
+useradd redis
+
+# Make directories
+mkdir -p $DOWNLOAD_DIR $CTRL_IIP_PREFIX /var/log/iip
+
+# Checkout ctrl_iip
+cd $DOWNLOAD_DIR
+curl -L https://github.com/lsst/ctrl_iip/archive/v2.1.tar.gz -o v2.1.tar.gz
+tar zxvf v2.1.tar.gz
+mv ctrl_iip-2.1 $CTRL_IIP_PREFIX/ctrl_iip
+
+# Install C++ dependency libraries
+# download boost 1.66
+cd $DOWNLOAD_DIR
+curl -L https://dl.bintray.com/boostorg/release/1.66.0/source/boost_1_66_0.tar.gz -o boost_1_66_0.tar.gz
+tar zxvf boost_1_66_0.tar.gz
+cd boost_1_66_0
+./bootstrap.sh --prefix=/opt/lsst
+./b2 install
+
+# download yaml-cpp
```

Comment:
Please check if yaml-cpp is available via rpm

Reply:
Version 0.5.1 is available via rpm, but we are using 0.5.3.

Reply:
Check to see if 0.5.3 is now available via RPM.  If it is, install the version.  Otherwise, use 0.5.3, and install it the way you've been doing it.